Automattic/jetpack
# JS Components: Add QRCode component

Fixes https://github.com/Automattic/jetpack/issues/23272

#### Changes proposed in this Pull Request:

* JS Components: Add ~QRPost~ QRCode component

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test with Storybook
```cli
cd ./projects/js-packages/storybook
```

```cli
pnpm install & pnpm storybook:dev
```

A browser tab should be opened with the `http://localhost:50240` address.

* Activate the QRPost story: `http://localhost:50240/?path=/story/js-packages-components-block-editor-qr-code--default`
* Play with the Addons tool to change component properties ~less the `value` one. It isn't possible to change it because it's internally handled by the store hooks.~
